Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.AllSatisfy.cs

```diff
@@ -108,11 +108,11 @@ public void Any_items_not_satisfying_inspector_should_throw()
 *Expected customer.Age to be less than 21, but found 21
 *Expected customer.Items to contain only items satisfying the inspector:
 *At index 0:
-*Expected item to be 3, but found 1
+*Expected item to be 3, but found 1 (which differs by -2)
 *At index 1:
-*Expected item to be 3, but found 2
+*Expected item to be 3, but found 2 (which differs by -1)
```

Comment:
For integers you may be right, but what about decimal values?
In those cases I think it does add value to see the difference when it's less than 10 (or even 1 for that matter).
So the real question is whether we run specific "filtering logic" per numeric type?

Reply:
> For integers you may be right, but what about decimal values?

There the details are important. 

> So the real question is whether we run specific "filtering logic" per numeric type?

I think we should. And it'll be relatively easy to do that since you have separate assertion classes per numeric type now.